tokio-postgres/src/lib.rs

```diff
@@ -180,12 +180,32 @@ mod transaction_builder;
 pub mod types;
 
 /// A convenience function which parses a connection string and connects to the database.
+/// 
+/// This method returns the client which will allow you to interact with the database and a connection
+/// object which is the one that performs the communication with the dabase. **It's important to use the
+/// connection object as shown in the below example, if it is ignored then the client object will not work
+/// and when a .wait is called it will await forever.**
```

Comment:
I've cleaned it up a bit.

```suggestion
/// This method returns the client object, which will allow you to interact with the database,
/// and a connection object, which performs the communication with the database.
///
/// **The connection object must be actively polled, which is usually done by spawning it into
/// it's own tokio task, as shown below. If this is not done, calls to the client object will hang forever.**
```